File: ProgAndPlay/Client_Interfaces/python/mission2.py
from constantList_KP4_1 import *
from pp import *

# Ouverture de la connexion avec le jeu
PP_Open()
# Récupération de la première unité
u = PP_GetUnitAt(MY_COALITION, 0)
# Récupération de la position de l'unité
pDepart = PP_Unit_GetPosition(u)
# Calcul de la coordonnée d'arrivée
pArrivee = PP_Pos()
pArrivee.x = pDepart.x - 927
pArrivee.y = pDepart.y + 513
# Ordonner à mon unité courante de se déplacer à la position calculée
PP_Unit_ActionOnPosition(u, MOVE, pArrivee)
# Fermer la connexion avec le jeu
PP_Close()

File: ProgAndPlay/Client_Interfaces/c-cpp/exemple/mission2.c
#include "PP_Client.h"
#include "../constantList_KP4.4.h"
#include "PP_Error.h"

// déplacer une unité à une position précise
int main (){
	// ouverture du jeu
	PP_Open();
	// récupérer la première unité
	PP_Unit u = PP_GetUnitAt(MY_COALITION, 0);
	// récupération des coordonnées de l'unité
	PP_Pos posDepart = PP_Unit_GetPosition(u);
	// calcul de la coordonnée d'arrivée
	PP_Pos posArrivee;
	posArrivee.x = posDepart.x - 927;
	posArrivee.y = posDepart.y + 513;
	// ordonner à mon unité courante de se déplacer
	PP_Unit_ActionOnPosition(u, MOVE, posArrivee);
	// fermer le jeu
	PP_Close();
}
